packages/angular/test/base/src/app/lazy/form/form.component.html

```diff
@@ -44,6 +44,20 @@
         <ion-input label="Input" formControlName="input2"></ion-input>
       </ion-item>
 
+      <ion-item>
+        <ion-input-otp id="touched-input-otp-number-test" formControlName="inputOtp" class="required">Input OTP (required)</ion-input-otp>
+      </ion-item>
+
+      <ion-item>
+        <ion-input-otp id="touched-input-otp-text-test" type="text" formControlName="inputOtpText" class="required">Input OTP Text (required)</ion-input-otp>
+      </ion-item>
+
+      <ion-button id="input-otp-touched" (click)="setOtpTouched()">Set Input OTP Touched</ion-button>
```

Comment:
Let's move this button to be under the first otp for clarity. I thought it was going to target the otp with the form control name of `inputOtpText`.

Reply:
Moved in https://github.com/ionic-team/ionic-framework/pull/30386/commits/4015ee4fea8cb6d6b17ef935aecfb7919137a09d